docs/advanced/cancelation.md

```diff
@@ -0,0 +1,27 @@
+# Cancelling
+
+If you have an async action sometimes you want to cancel a previous observable if the action is dispatched again. We can handle this with the `takeUntil` operator.
+
+```TS
+import { State, Action, Actions, ofAction } from '@ngxs/store';
+import { tap } from 'rxjs/operators';
+
+@State<ZooStateModel>({
+  defaults: {
+    animals: []
+  }
+})
+export class ZooState {
+  constructor(private animalService: AnimalService) {}
+
+  @Action(FeedAnimals)
+  get({ getState, setState, dispatch }, { payload }) {
+    return this.animalService.get(payload).pipe(
+      tap((res) => {
+        setState(res)
+      }),
+      takeUntil(this.actions.pipe(ofAction(RemoveTodo)))
```

Comment:
Where does `this.actions` come from? Typescript complains:
> Property 'actions' does not exist on type 'ZooState'

Reply:
import it from @ngxs/store

Reply:
ah I didn't inject it in the example. will do